resources/js/src/app/components/itemList/SearchSuggestionItems.vue

```diff
@@ -63,14 +63,15 @@ export default {
     {
         getDisplayName(itemData)
         {
-            let displayName = this.$options.filters.itemName(itemData);
+            const search = this.autocompleteSearchString.split(" ")
```

Comment:
```suggestion
            const search = this.autocompleteSearchString.split(/\s+/)
```

so trennst du an allen Whitespaces, also auch mehrere Leerzeichen hintereinander oder Tabs.